### ISSUE TITLE
Webtorrent unable to find peers when connecting to private tracker

### ISSUE THREAD

**Issue by tab1293** [feross/webtorrent#1032]
**What version of WebTorrent?**
^0.98.3

**What operating system and Node.js version?**
OS X 10.11.3 with Node v6.4.0

**What did you expect to happen?**
Expected peers to be found and files to be downloaded

**What actually happened?**
According to the regular output, no peers were added and no pieces were downloaded.

Here is the debug output logged from just running `webtorrent download somefile.torrent`
https://gist.github.com/tab1293/76f500433623035c4cd16f85e89d7910

The log file looks like peer wire connections are being established and messages are being passed between me and other peers. I can't find any glaring differences in a log file from a public tracker which succeeds in downloading. Can anyone tell from the log file what might be failing?

I'm not too keen on the implementations of private trackers. Could the tracker and other peers potentially be blocking webtorrent's peer id?

**Comment by tab1293** [comment]
@feross any ideas on what may be causing this to happen?

**Comment by zephrax** [comment]
Is #1054 related?

**Comment by tab1293** [comment]
@zephrax I don't believe that issue is related. I am not attempting to seed the file or download it within a browser.

I am unable to download a torrent from a private tracker while using the webtorrent command line program or the nodejs module. Any other information I can provide besides the debug output that could be useful? Maybe a pcap file or something?
